js/rpg_managers/ImageManager.js

```diff
@@ -128,3 +136,175 @@ ImageManager.isBigCharacter = function(filename) {
 ImageManager.isZeroParallax = function(filename) {
     return filename.charAt(0) === '!';
 };
+
+
+ImageManager.reserveAnimation = function(filename, hue, reserveId) {
```

Comment:
In 141, 145, you use `reserveId`.
In 149, 153, and others, you use `reservationId`. 
Can you unify?